packages/fluttium/lib/src/tester.dart

```diff
@@ -169,8 +170,7 @@ class Tester {
 
   /// Wait for the semantics tree to be fully build.
   Future<void> ready() async {
-    while (_binding.pipelineOwner.semanticsOwner == null ||
-        _binding.pipelineOwner.semanticsOwner!.rootSemanticsNode == null) {
+    while (_semanticsOwner?.rootSemanticsNode == null) {
```

Comment:
Okay makes sense.

I just wrote the same logic in a shorter way.

Before:
```
_binding.pipelineOwner.semanticsOwner == null ||
        _binding.pipelineOwner.semanticsOwner!.rootSemanticsNode == null
```

```suggestion
    while (_semanticsOwner!.rootSemanticsNode == null) {
```